tox.ini

```diff
@@ -37,6 +37,7 @@ commands = {posargs}
 #Skip PEP257 violation.
 [flake8]
 ignore = D100,D101,D102,D103,D104,D105,D200,D203,D202,D204,D205,D208,D400,D401
+max-line-length = 100
```

Comment:
Please leave default to max line length to 79 to match w/ Mistral. As I explained in Slack, this is an extension project for Mistral and I have been following the coding guidelines for Mistral. Sorry if this doesn't make any sense.